samples/traveling-salesman/README.md

```diff
@@ -0,0 +1,69 @@
+---
+page_type: sample
+languages:
+- python
+author: Maximilian Lucassen
+ms.author: t-mlucassen@microsoft.com
+ms.date: 4/9/2021
+products:
+- azure-quantum
+- azure-qio
+description: "Solve the traveling salesman problem with Azure Quantum optimization service"
+urlFragment: azure-quantum.traveling-salesman
+---
+
+# Solving the traveling salesman problem with the Azure Quantum optimization service
+
+## Introduction
+
+This sample provides a comprehensive walkthrough of the traveling salesman problem, from problem definition to formulation of penalty functions and finally solving the problem using the Azure Quantum Optimization Service.
+
+By working through this sample, you will learn:
+
+- What the traveling salesman problem is and why it is important
+- How to represent problem terms mathematically
+- How to build penalty functions to represent problem constraints
+- How to build an objective function to minimize the makespan
+- How to transform these mathematical functions into code using the Azure Quantum Optimization Python SDK
+- How to submit problem terms to Azure Quantum
+- How to interpret the results
+- How to validate the results
+- How to tune the solver parameters
```

Comment:
Please update these points to accurately reflect what points are covered in the sample

Reply:
(see the other samples for examples)